toolkit/scripts/imggen.mk

```diff
@@ -5,7 +5,13 @@ $(call create_folder,$(IMAGEGEN_DIR))
 
 # Resources
 config_name              = $(notdir $(CONFIG_FILE:%.json=%))
-config_other_files       = $(if $(CONFIG_FILE),$(call shell_real_build_only, find $(CONFIG_BASE_DIR)))
+CONFIG_BASE_DIR          = $(dir $(CONFIG_FILE:%.json=%))
+config_other_files      := $(shell grep -E '.json|.sh' $(CONFIG_FILE)| sed 's/\"Path\"\://' | tr ", \n" " " | tr "\"" " " | xargs)
```

Comment:
Is this going to behave ok with `CONFIG_FILE=""`? Need to make sure that we don't end up with an error message or something in `config_other_files` which Make will then try to find real files for.